services/dynamic-sidecar/tests/unit/test_api_compose.py

```diff
@@ -0,0 +1,158 @@
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+
+import json
+from typing import Any, Dict
+
+import pytest
+from async_asgi_testclient import TestClient
+from faker import Faker
+
+DEFAULT_COMMAND_TIMEOUT = 10.0
+
+
+@pytest.fixture
+def compose_spec() -> str:
+    return json.dumps(
+        {
+            "version": "3",
+            "services": {"nginx": {"image": "busybox"}},
```

Comment:
maybe have a bit more complex docker-compose here? you could have actually an empty one, one with 1 service, one with 2 (basically you could take one of @mguidon ?) and parametrize the tests.

Reply:
Was thinking of adding them with the integration testing in the director-v2 with a proper full workflow.